index.js

```diff
@@ -32,6 +40,16 @@ function gulpAmdCheck(options) {
         file.contents = new Buffer(output.optimizedContent);
       }
 
+      if (options.errorOnUnusedDependencies) {
+        if (output.results.length) {
+          var unusedDependencies = output.results[0].unusedDependencies;

```

Comment:
Yes, working on it now

On Sat, 25 Apr 2015 at 12:15 Mehdi Shojaei <notifications@github.com> wrote:

> In index.js
> <https://github.com/mehdishojaei/gulp-amdcheck/pull/4#discussion_r29099577>
> :
>
> > @@ -32,6 +40,16 @@ function gulpAmdCheck(options) {
> >          file.contents = new Buffer(output.optimizedContent);
> >        }
> >
> > +      if (options.errorOnUnusedDependencies) {
> > +        if (output.results.length) {
> > +          var unusedDependencies = output.results[0].unusedDependencies;
>
> Please check other results elements too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mehdishojaei/gulp-amdcheck/pull/4/files#r29099577>.
>